app/page/authorization/transaction.tsx

```diff
@@ -270,71 +277,59 @@ function UserOperationConfirmation(props: {
+        <h2 className="py-[12px]">From</h2>
+        <div className="flex gap-[12px] items-center">
+          <Wallet />
+          <div className="w-[322px] py-[9.5px]">
+            <h3>Jesse's wallet</h3>
+            <h4 className="text-[#989898] ">{userOp.sender}</h4>
+          </div>
         </div>
-      </div>
-      <div>
-        <h1>Paymaster Option</h1>
-        {paymentOptions.map((o, i) => {
-          const id = i.toString()
-          const isSelected = o.name === paymentOption.name
-          return (
-            <div key={i}>
-              <input
-                type="checkbox"
-                id={id}
-                name={o.name}
-                checked={isSelected}
-                disabled={paymentCalculating || isSelected || userOpEstimating}
-                onChange={() => setPaymentOption(o)}
-              />
-              <label htmlFor={id}>{o.name}</label>
-            </div>
-          )
-        })}
-      </div>
-      <div>
-        <h1>Transaction Cost</h1>
-        <p>
-          Estimated gas fee:{" "}
-          {userOpEstimating || paymentCalculating
-            ? "Estimating..."
-            : `${ethers.formatEther(userOp ? userOp.calculateGasFee() : 0n)} ${
-                ETH.symbol
-              }`}
-        </p>
-        <p>
-          Expected to pay:{" "}
-          {userOpEstimating || paymentCalculating
-            ? "Calculating..."
-            : `${ethers.formatUnits(
-                payment.tokenFee,
-                payment.token.decimals
-              )} ${payment.token.symbol}`}
-        </p>
-      </div>
-      <div className="mt-1">
-        <button
+        <h2 className="py-[12px]">To</h2>
+        <div className="flex gap-[12px] items-center">
+          <Wallet />
+          <div className="py-[16px] w-[322px]">
+            <h3>{props.tx.to}</h3>
+          </div>
+        </div>
+      </section>
+      <Divider />
+      <section className="py-[16px] text-[16px]">
+        <h2 className="py-[8px]">Est. gas fee</h2>
+        <div className="flex gap-[12px] py-[16px]">
+          <Gas />
+          <p>
+            {userOpEstimating || paymentCalculating
+              ? "Estimating..."
+              : `${ethers.formatEther(
+                  userOp ? userOp.calculateGasFee() : 0n
+                )} ${ETH.symbol}`}
+          </p>
+        </div>
+      </section>
+
+      <div className="py-[22.5px] flex justify-between gap-[16px] text-[18px] font-semibold">
+        <Button
+          disabled={userOpResolving}
+          onClick={rejectUserOperation}
+          text="Cancel"
+          variant="white"
+        />
+        <Button
           disabled={paymentCalculating || userOpEstimating || userOpResolving}
-          onClick={sendUserOperation}>
-          Send
-        </button>
-        <button disabled={userOpResolving} onClick={rejectUserOperation}>
-          Reject
-        </button>
+          onClick={sendUserOperation}
+          text="Send"
```

Comment:
It should be `Confirm` text.

Reply:
[btn text should be Confirm](https://github.com/pilagod/waallet-extension/pull/152/commits/16d5ad79d2dbdd825a46462e609afe9aad520818)